README.md

```diff
@@ -16,8 +16,17 @@ To run the image, simply invoke a shell in the container and start squid in the
 are interested in:
 
 
+```bash
```

Comment:
suggest a table with all the config values


To run the image, either specify the configuration you need or run in an interactive shell


| Configuration File | Description |
|:------------|-------------|
| **`squid.conf.forward`** | explicit forward proxy mode intercepts HTTP traffic and uses CONNECT for https. (`default`) |
| **`squid.conf.cache`** | squid proxy which will cache upstream content   |
| **`squid.conf.https_port`** | forward proxy with HTTPS listener for squid  |
| **`squid.conf.basicauth`** | enable basic authentication with username/password|
| **`squid.conf.allow_domain`** | forward proxy that allows access to specific domain  |
| **`squid.conf.intercept`** | intercept and inspect SSL Traffic |

```bash
# detached background run (uses squid.conf.forward)
docker run -d -p 3128:3128 docker.io/salrashid123/squidproxy

# alternative configuration file
docker run -it -p 3128:3128 -e CONF=squid.conf.intercept docker.io/salrashid123/squidproxy
```

```bash
# interactive
docker run -it -p 3128:3128 docker.io/salrashid123/squidproxy /bin/bash
```